src/client/styles/_topbar.scss

```diff
@@ -67,6 +67,24 @@
     margin-right: 10px;
 }
 
+#run-btn .main-text {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+#run-btn .main-text .inactive {
```

Comment:
Sass allows you to nest styles, so you can write code like this:

```
#run-btn .main-text {
    display: flex;
    justify-content: center;
    align-items: center;

    .inactive {
        position: absolute;
        display: hidden;
        transition: transform 0.4s ease-out;
    }

    .active {
        ...
    }
}
```

Reply:
That's awesome--I've never used Sass before, but I've read up on it and I think I know what I'm doing now :) Thanks!